tests/test_ast_package_symbol_extractor.py

```diff
@@ -0,0 +1,78 @@
+from symbol_exporter.ast_package_symbol_extractor import DirectorySymbolFinder
+from symbol_exporter.ast_symbol_extractor import SymbolType
+from symbol_exporter.db_access_model import make_json_friendly
+
+expected = {
```

Comment:
This is expected final symbol table for test package numpy.
Please review for any omissions.

Reply:
Actually just realized I want to add another case.

We should add the star imports that are imported indirectly into the namespace through relative star imports.

for example:
file `numpy.__init__.py` contains
`from .core import *`

file `numpy/core.__init__.py` contains
`from requests import *`

We should add `requests` to the set of star imports for symbol "numpy.*"

Reply:
done